Server/fixtures/mobile.js

```diff
@@ -129,43 +154,6 @@ exports.createFixtures = async () => {
           password: bcrypt.hashSync("user", 12),
           mailVerification: true,
           notificationsId: notification2.id,
-          itemOpinions: {
```

Comment:
put back the fixtures for the mobile

Reply:
how is this resolved ? where is the itemOpinion and the new user ?

Reply:
I put it back